.github/workflows/verify.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on: [push]
+
+jobs:
+  verify:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+
+    - name: generate
+      uses: actions-contrib/go@v0.1.0
+      with:
+        args: run pkg/generator/step-metadata.go
+
+    - name: format
+      uses: actions-contrib/go@v0.1.0
+      with:
+        args: fmt
```

Comment:
my understanding is that we need to check that go fmt does not print any text, then the code is propertly formatted. otherwise it has some complaints and we should fail the pipeline.

Reply:
needs to be changed to `go fmt ./...`, that will format all files. Changed files will be detected by `git diff`.